client/state/posts/utils.js

```diff
@@ -91,3 +98,60 @@ export function mergeIgnoringArrays( object, ...sources ) {
 		}
 	} );
 }
+
+/**
+ * Given a post object, returns a normalized post object
+ *
+ * @param  {Ojbect} post Raw edited post object
+ * @return {Object}      Normalized post object
+ */
+export function normalizePost( post ) {
```

Comment:
Should this be `normalizeEditedPost`? Since we treat posts distinctly between display and editing.